src/core/public/chrome/ui/header/header.tsx

```diff
@@ -195,13 +196,15 @@ export function Header({
                   aria-controls={navId}
                   ref={toggleCollapsibleNavRef}
                 >
-                  <EuiIcon
-                    type="menu"
-                    size="m"
-                    title={i18n.translate('core.ui.primaryNav.menu', {
+                  <EuiToolTip
+                    content={i18n.translate('core.ui.primaryNav.menu', {
                       defaultMessage: 'Menu',
                     })}
-                  />
+                    delay="long"
+                    position="bottom"
+                  >
+                    <EuiIcon type="menu" size="m" />
```

Comment:
Let's add `aria-label` to the `EuiIcon` with the tooltip's content.

Reply:
Can we make sure this is necessary? The icon is not focusable (its in a button), so I don't think the aria label matters in this case.

Agree for other cases it may be the right decision.

Reply:
Honestly, does it make more sense for the tooltip to be around the button and match the button aria-label?

Reply:
You are right; there is a button around it. The content need not have.